src/main.py

```diff
@@ -0,0 +1,242 @@
+# Standard Python Libraries
```

Comment:
i did run `black src/main.py` with following version
```bash
> black --version
black, 22.6.0 (compiled: no)
Python (CPython) 3.10.6
```
and the reformat is the following
```diff
diff --git a/src/main.py b/src/main.py
index a8e1476..00a6314 100644
--- a/src/main.py
+++ b/src/main.py
@@ -44,7 +44,7 @@ class Interact:
     ) -> None:
         """Copies a file."""
         original = pathlib.PosixPath(path_to_source) / file_name
-        target = pathlib.PosixPath(working_dir)/ folder_name / file_name
+        target = pathlib.PosixPath(working_dir) / folder_name / file_name
         shutil.copyfile(original, target)
 
     def search_file(search_word: str, file_name: str) -> bool:
@@ -160,7 +160,9 @@ class Settings:
         class_file = "".join([class_name, ".tex"])  # main LaTeX file.
         path = pathlib.PosixPath(Settings.get_path_to_source(path_to_json), class_file)
         for dep in dep_list:
-            if dep != class_file and not Interact.search_file(pathlib.PosixPath(dep).stem, path):
+            if dep != class_file and not Interact.search_file(
+                pathlib.PosixPath(dep).stem, path
+            ):
                 return False
         return True
 
@@ -202,7 +204,9 @@ class Project:
         if Project.check_init(self):
             dep_list = Settings.get_class_dep(path_to_json, self.latex_class)
             for dep in dep_list:
-                Interact.copy_file(self.working_dir,Project.path_to_source,dep,self.project_name) # Copy required TeX files
+                Interact.copy_file(
+                    self.working_dir, Project.path_to_source, dep, self.project_name
+                )  # Copy required TeX files
             if images:
                 Interact.make_folder(
                     os.path.join(self.working_dir, self.project_name), "images"
@@ -239,4 +243,5 @@ class Project:
         except:
             traceback.print_exc()
 
-# Interface
\ No newline at end of file
+
+# Interface
```

Reply:
and with `flake8`,

```bash
> flake8 --version
3.8.4 (mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.8.13 on Linux
```
gives me
```rust
> flake8 src/main.py
src/main.py:9:1: F401 'readline' imported but unused
src/main.py:13:1: F401 'rich' imported but unused
src/main.py:16:1: F401 'rich.prompt.Prompt' imported but unused
src/main.py:17:1: F401 'rich.prompt.Confirm' imported but unused
src/main.py:18:1: F401 'rich.markdown.Markdown' imported but unused
src/main.py:19:1: F401 'rich.console.Console' imported but unused
src/main.py:78:80: E501 line too long (83 > 79 characters)
src/main.py:95:80: E501 line too long (82 > 79 characters)
src/main.py:111:80: E501 line too long (82 > 79 characters)
src/main.py:129:80: E501 line too long (81 > 79 characters)
src/main.py:143:80: E501 line too long (85 > 79 characters)
src/main.py:152:80: E501 line too long (98 > 79 characters)
src/main.py:161:80: E501 line too long (87 > 79 characters)
src/main.py:203:80: E501 line too long (88 > 79 characters)
src/main.py:208:80: E501 line too long (84 > 79 characters)
src/main.py:223:13: E722 do not use bare 'except'
src/main.py:232:80: E501 line too long (82 > 79 characters)
src/main.py:233:80: E501 line too long (80 > 79 characters)
src/main.py:243:9: E722 do not use bare 'except'
```

Reply:
if you have the same messages, could you please fix the `./src/main.py` file? :yum: